Project-3/README.md

```diff
@@ -8,70 +8,431 @@ For any question involving the use of Protege, please be sure to import:
-```
+
   (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  
+   Answer:  "Sally participates in having at least one arm on Tuesday" and "Sally participates in having no arms on Wednesday"
+   
+           Sally is instance_of object
+           Tuesday is instance_of one-dimensional temporal region 
+           Wednesday is an instance of one-dimensional temporal region 
+           Arm is an instance of fiat object part
+           “Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
+           “Sally participates in having no arms on Wednesday” is an instance of occurrent
+           We don't know whether Tuesday precedes Wednesday. 
+   
   (b) Every liver has some cell as part at all times it exists.
+  
+   Answer: Liver has_part_at_all_times Cell
+   
+   Liver is a class in Uberon (UBERON_0002107), which is a subclass of material entity. 
+   We have already had Cell in the Ontology. 
+  
   (c) John was a child, then an adult, then a senior. 
+  
+  Answer:  John participates in childhood precedes John participates in adulthood which precedes John participates in seniorhood
+  
+     John is an instance_of object
+     John's childhood is an instance_of occurrent.
+     John's adulthood is an instance_of occurrent.
+     John's seniorhood is an instance_of occurrent.
+  
   (d) Goofus and Gallant are married at each point in a three year span. 
-```
+  
+  Answer: Goofus participates in marriage at t1  and  Galland participates in marriage at t1.
+  
+   Goofus is an instance_of object
+   Gallant is an instance_of object
+   Marriage is an instance_of occurrent
+   “Three years span 1” is an instance_of one-dimensional temporal region.
+   Goofus and Gallant may be not married to each other.
 
 4. Using the language of First-Order Logic, represent the following natural language expressions; you are welcome to introduce new terms where needed: 
-```
+
   (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  
+   ∃x (Tx ∧ ∃y (Hsy∧Ay)) ∧ ∃x (Wx ∧ ~∃y(Hsy∧Ay))
+    T: Tuesday
+    H: has
+    A: arm
+    W: Wednesday
+    s: Sally
+ 
   (b) Every liver has some cell as part at all times it exists.
+   
+   ∀x∃y(Lx→Cy∧Pyx)
+     L: liver
+     C: cell
+     P: part of
+  
   (c) John was a child, then an adult, then a senior. 
+  
+  j = John
+  E (x, y) = being earlier than
+  C (x, t) = being a child at t
+  A (x, t) = being an adult at t
+  S (x, t) = being a senior at t
+  
+  ∃t1∃t2∃t3 (C (j, t1) ∧ A (J, t2) ∧ S(J, t3) ∧ E (t1, t2) ∧ E (t2, t3)) 
+  
   (d) Goofus and Gallant have been married for three years; for each day of that span, it is true to assert they are married. 
-```
+     
+     M(x, t) = being married at t
+     Y(t) = belongs to 3 year span 1
+     g1 = Goofus
+     g2 = Gallant
+     D(t) = t is a day
+      
+     ∀t(D(t) ∧ Y(t)→(M(g1,t) ∧ M(g2,t)))
```

Comment:
The predicates don't make sense...

Also, this says at most that Goofus and Gallant are married one day in a three year span...